### PR TITLE
Fix Loading of List User in Admin

### LINKED ISSUE #1287: Fix Loading of List User in Admin
**Actual Behaviour**
Currently the users loaded in admin panel.

**Expected Behaviour**
Fix it.

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by harsh-jindal**
Fixes #1287 

Changes: Made changes in aaa/GetUsers.java

### CODE REVIEW

**Comment by harsh-jindal** [comment]
@Orbiter @akshatnitd @fragm3 Please review.

**Comment by fragm3** [comment]
I tried to make it work, a few days back. But the problem is, it's working on localhost, without any changes. How can we test if it can work on a real server instance? @akshatnitd

**Comment by harsh-jindal** [comment]
@fragm3 It doesn't work for me on my localhost. Also it's strange how it works for you because if you pass only access token as parameter you shall get `Bad request. No parameter present` as per the current code. Also you get java.array.outofbound exception because of page =0.

**Comment by fragm3** [comment]
@harsh-jindal I tried with correct query parameters, which are specified in the Servlet file.

**Comment by harsh-jindal** [comment]
@fragm3 Then it would be best if we have some way to check it in on real server instance as said by you.

File: src/ai/susi/server/api/aaa/GetUsers.java [hunk 1]
```diff
@@ -46,13 +46,6 @@ public JSONObject getDefaultPermissions(UserRole baseUserRole) {
 
     @Override
     public ServiceResponse serviceImpl(Query call, HttpServletResponse response, Authorization rights, final JsonObjectWithDefault permissions) throws APIException {
-        if (call.get("getPageCount", false) == false
```
Comment (akshatnitd) [inline src/ai/susi/server/api/aaa/GetUsers.java]: This check is important. Do not remove it.

File: src/ai/susi/server/api/aaa/GetUsers.java [hunk 2]
```diff
@@ -74,7 +67,7 @@ public ServiceResponse serviceImpl(Query call, HttpServletResponse response, Aut
             result.put("message", "Success: Fetched count of users");
             return new ServiceResponse(result);
         } else {
-            int page = call.get("page", 0);
+            int page = call.get("page", 1);
```
Comment (akshatnitd) [inline src/ai/susi/server/api/aaa/GetUsers.java]: Default page should be 0

**Pull request closed without merging**